website/content/docs/k8s/architecture.mdx

```diff
@@ -0,0 +1,76 @@
+---
+layout: docs
+page_title: Consul on Kubernetes Architecture
+description: >-
+ A high level overview of Consul on Kubernetes Architecture 
+---
+
+
+# Architecture
+
+Consul runs on Kubernetes with the same
+[architecture](/docs/architecture)
+as other platforms. There are some benefits Kubernetes can provide
+that eases operating a Consul cluster and we document those below. The standard
+[production deployment guide](https://learn.hashicorp.com/consul/datacenter-deploy/deployment-guide) is still an
+important read even if running Consul within Kubernetes.
+
+Each section below will outline the different components of running Consul
+on Kubernetes and an overview of the resources that are used within the
+Kubernetes cluster.
+
+## Server Agents
+
+The server agents are run as a **StatefulSet**, using persistent volume
```

Comment:
```suggestion
The server agents are run as a **StatefulSet**, using persistent volume
```
I think that because the Kubernetes keywords are configuration settings (or appear to be based on my Google search), we should use code font and reposition how we reference them to make this clear. 

For example, we might want to rephrase this sentence to something like:

Consul server agents operate as `StatefulSets` types and use persistent . . . "

More precise, no?

Reply:
Thanks!